File: PHP/Card.php
<?php
class Card extends Payment{
  public $numberCard ;
  public $cvv;
  public $expiration;
  public $amount;

  public function __construct ($id, $numberCard, $cvv ,$expiration, $amount){
    parent :: __construct($id);
    $this->numberCard = $numberCard;
    $this->cvv = $cvv;
    $this->expiration = $expiration;
    $this->amount = $amount;
  }
}
?>

File: Python/Card.py
from Payment import Payment


class Card(Payment):
    numberCard = int
    cvv = int
    expire = str
    amount = float

    def __init__(self, numberCard, cvv, expire, amount):
        super().__init__()
        self.amount = amount
        self.cvv = cvv
        self.expire = expire
        self.numberCard = numberCard
